trustd/src/db.rs

```diff
@@ -67,12 +68,17 @@ impl Run {
         let db_dir = work_dir.join("postgres");
         let data_dir = work_dir.join("data");
         create_dir_all(&data_dir)?;
+        let db_conf = HashMap::from([(
+            "shared_preload_libraries".to_string(),
+            "pg_stat_statements".to_string(),
+        )]);
         let settings = postgresql_embedded::Settings {
             username: self.database.username.clone(),
             password: self.database.password.clone(),
             temporary: false,
             installation_dir: db_dir.clone(),
             timeout: Some(Duration::from_secs(30)),
+            configuration: db_conf,
```

Comment:
In a case like that, you might want to consider using the same name for the variable as for the field. In which case you can collapse this into:

```rust
let configuration = ();
let settings = Settings {
  configuration,
};
```

I am not sure it makes sense here. Just a note.

Reply:
I originally did that but lint complained ... will investigate again.

Reply:
works now, thx